src/api/spec/policies/report_policy_spec.rb

```diff
@@ -0,0 +1,169 @@
+require 'rails_helper'
+
+RSpec.describe ReportPolicy, type: :policy do
+  subject { ReportPolicy }
+
+  permissions :show? do
+    context 'when the current user is the owner of the report' do
+      let(:user) { create(:confirmed_user) }
+      let(:record) { create(:report_for_comment, user: user) }
+
+      it { is_expected.to permit(user, record) }
+    end
+
+    context 'when the current user is an admin' do
+      let(:user) { create(:admin_user) }
+      let(:record) { create(:report_for_comment) }
+
+      it { is_expected.to permit(user, record) }
+    end
+
+    context 'when the current user is a regular user' do
+      let(:user) { create(:confirmed_user) }
+      let(:record) { create(:report_for_comment) }
+
+      it { is_expected.not_to permit(user, record) }
+    end
+  end
+
+  permissions :create? do
+    before do
+      Flipper.enable(:content_moderation, user)
+    end
+
+    context 'when the user does not have the content_moderation active' do
```

Comment:
I don't think we need to spec this. There is no logic involved, just makes it harder to rip out once we roll this out...

Reply:
Okay